plugins/jobtype/src/azkaban/jobtype/HadoopSecureSparkWrapper.java

```diff
@@ -0,0 +1,157 @@
+/*
```

Comment:
We should add private.properties file as in https://github.com/azkaban/azkaban-plugins/tree/master/plugins/jobtype/jobtypes